chapters/en/Unit 2 - Convolutional Neural Networks/yolo.mdx

```diff
@@ -0,0 +1,73 @@
+# YOLO
+
+## Evolution of YOLO
+
+So far, we have seen the basic characteristics of YOLO and how it allows it to perform highly accurate and fast predictions. YOLOv1 was released in 2015 and until today multiple versions have been released. YOLOv1 was groundbreaking in terms of accuracy and speed. However, it suffered from poor localization and detection of multiple objects in a small image area.
+
+### **YOLOv2 (2016)** 
+
+After a year of releasing the first version, YOLOv2[1] came out. The improvements focused on both accuracy and speed but also dealt with the localization problem. Many improvements were made (image1). First, YOLOv2 replaced YOLOv1’s backbone architecture with Darknet-19, which is a variant of the Darknet architecture. Darknet-19 is lighter than the previous version’s backbone and it consists of 19 convolutional layers followed by max-pooling layers. This led to YOLOv2 having the ability to capture more information. Also, it included batch normalization to all convolutional layers removed the dropout layers dealing with the overfitting problem, and increased the mAP. It also introduced anchor boxes, which added prior knowledge to the detected width and height of the detected boxes (specifically they used). Additionally, to deal with the poor localization YOLOv2 predicted the class and objects for every anchor box and as previously grid (now 13x13). So we have a maximum (for 5 anchor boxes) of 13x13x5 = 845 boxes.
+
+### **YOLOv3 (2018)**
+
+YOLOv3[2] again improved significantly the detection speed and accuracy by replacing the Darknet-19 architecture with the more complex but efficient Darknet-53. Also, it dealt more with the localization problem by using three different scales for object detection (13x13, 26x26, and 52x52 grids). This helped find objects with different sizes in the same area. This increased the bounding boxes to: 13 x 13 x 3 + 26 x 26 x 3 + 52 x 52 x 3 = 10,647. The Non-Maximum Suppression (NMS) was still used to filter out redundant overlapping boxes.
+
+### **YOLOv4 (2020)**
+
+Back in 2020, YOLOv4[3] became one of the best detection models in terms of speed and accuracy. The authors changed again the backbone architecture to the faster and better accuracy CSPDarknet53[4]. An important improvement of this version was the optimization for efficient resource utilization, making it suitable for deployment on various hardware platforms, including edge devices. Also, It included a number of augmentations before training that further improved the model prediction accuracy on custom data.
```

Comment:
what exact accuracy? by custom data you mean new dataset?

Reply:
_**what exact accuracy?**_ In object detection benchmarks like COCO dataset

 **_by custom data you mean new dataset?_** Yes, but I will change that and mention that it helps in the generalization of the model.